.travis.yml

```diff
@@ -61,7 +61,7 @@ before_install:
 - export PATH=$HOME/.local/bin:$PATH
 - ./tools/install-stack.sh
 - ./tools/install-flatbuf.sh
-- export CLASSPATH=$CLASSPATH:$HOME/flatbuffers/java
+- export CLASSPATH=$CLASSPATH:./flatbuffers/java
```

Comment:
Likewise, we should not use relative `$CLASSPATH`:
```
export CLASSPATH=$CLASSPATH:`pwd`/flatbuffers/java
```

Reply:
Do you want to fix this one too (otherwise, I can do it; just want to be sure you're ok with the suggestion)?

Reply:
I actually fixed this one too, but I mistyped the file name when I committed...
Sorry about that. Pushed it now.